bindings/rust/s2n-tls/Cargo.toml

```diff
@@ -35,3 +36,4 @@ checkers = "0.6"
 # cargo package / cargo publish, as those commands do not respect the version pin
 # in downstream dev-dependencies (in s2n-tls-sys, in this case)
 jobserver = "=0.1.26"
+cfg-if = "1"
```

Comment:
What is this new dependency for?

Reply:
Testing related. It can prob be removed since we dont want to test the Disabled case

```
cfg_if::cfg_if! {
            if #[cfg(feature = "fips")] {
                assert_eq!(init::fips_mode().unwrap(), FipsMode::Enabled);
            } else {
                assert_eq!(init::fips_mode().unwrap(), FipsMode::Disabled);
            }
}
```